ci/Jenkinsfile.tests-e2e

```diff
@@ -249,3 +249,11 @@ def getDefaultTestScopeFlag() {
     return '-m=critical'
   }
 }
+
+def getTestStageTimeout() {
+  if (JOB_NAME == "status-desktop/systems/linux/x86_64/tests-e2e") {
```

Comment:
arent we using the same job everywhere?  How we understand which is nightly run, PR run or manually started job?

Reply:
I copy-pasted it from `getDefaultTestScopeFlag()`, but I can also rely on it, so if test scope flag is `-m=critical` - set timeout to 30 min.